build.sh

```diff
@@ -84,8 +85,8 @@ apt-get -q update
 apt-get -q -y --purge install ubuntu-standard casper lupin-casper \
   laptop-detect os-prober linux-generic
 
-#
-dpkg -i meilix-metapackage*.deb
+# TODO: Update melix-metapackage to work with Xenial
+#dpkg -i meilix-metapackage*.deb
```

Comment:
Yeah, I can see that. But the packages must be updated because they're not made for `xenial`. A simple rebuild and change of the metadata can fix this.